### PR TITLE
Add a note to the PULL_REQUEST_TEMPLATE about PR titles

### PR DESCRIPTION
**Pull request by asmeurer**
Hopefully this will stem the tide of "Fixes #NNNN" PR titles.

Note, I've intentionally not hard line-wrapped the text, as GitHub already soft wraps it.

### CODE REVIEW

**Comment by asmeurer** [comment]
Suggestions for improving the text are welcome.

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1 +1,3 @@
+<!-- Please give this pull request a descriptive title. Pull requests with descriptive titles are more likely to receive reviews. Describe what you changed! A title that only references an issue number is not descriptive. -->
+
 <!-- If this pull request fixes an issue please indicate which issue by typing "Fixes #NNNN" below.>
```
Comment (smichr) [inline PULL_REQUEST_TEMPLATE.md]: The last line should end as `-->`.

Comment (asmeurer) [inline PULL_REQUEST_TEMPLATE.md]: Fixed.

**Comment by smichr** [comment]
> requested a review from smichr 5 days ago

Just out of curiousity, where do these requests go?

**Comment by smichr** [comment]
Thanks, Aaron, for updating this.

**Comment by asmeurer** [comment]
It's on the top right of the PR page. I think you can also filter the PR list by requested reviews.

**Comment by asmeurer** [comment]
By the way, I have an email filter set up for review requests. I tend to not see most @mentions to me since there are so many of them and I can't read all my emails, but I will make sure to look at any review request, so feel free to use that to ping me on a PR.

**Pull request merged**